addon/services/flash-messages.js

```diff
@@ -68,6 +68,13 @@ export default Service.extend({
     return this;
   },
 
+  getFlashObject() {
```

Comment:
@panthony trying to wrap this up today. I like `peekLast` and `peekFirst` but what about a specific index?  Or should we not worry about that?

I kind of like `getFlashObject` because it implies that it is chainable and returns the flash object we just added.  If someone wants to get a specific index, they can access the queue from the service directly.